templates/admin.html

```diff
@@ -18,5 +18,22 @@ <h2>Home page content</h2>
 		<br />
 	</form>
 </section>
+<section>
+	<h2>Print All ID Cards</h2>
+	<form action=/cards/all>
+		<h3>Select semester for which to generate ID cards:</h3>
+		<select name="semester" size="{{ semesters|length }}">
```

Comment:
I am inclined to say make this a drop-down menu (i.e. no size) since the number of semesters will continue to increase, but people will probably only want to print one semester's ID cards.